packages/internals/src/schemaEngineCommands.ts

```diff
@@ -129,25 +129,25 @@ export async function createDatabase(connectionString: string, cwd = process.cwd
 
     if (e.stderr) {
       const logs = parseJsonFromStderr(e.stderr)
-      const error = logs.find((it) => it.level === 'ERROR' && it.target === 'migration_engine::logger')
+      const error = logs.find((it) => it.level === 'ERROR' && it.target === 'schema_engine::logger')
```

Comment:
Btw this is already `schema_engine` and not `migration_engine` since https://github.com/prisma/prisma-engines/pull/3844 (Prisma 4.13).

This functionality is likely not covered by tests on TS side since this didn't fail any tests.